lib/dataloader.ex

```diff
@@ -156,12 +161,32 @@ defmodule Dataloader do
         end)
         |> Map.new()
 
-      %{dataloader | sources: sources}
+      updated_dataloader = %{dataloader | sources: sources}
+
+      emit_stop_event(id, start_time_mono, updated_dataloader)
+
+      updated_dataloader
     else
       dataloader
     end
   end
 
+  defp emit_start_event(id, system_time, dataloader) do
+    :telemetry.execute(
+      [:absinthe, :middleware, :dataloader, :start],
+      %{system_time: system_time},
+      %{id: id, dataloader: dataloader}
+    )
+  end
+
+  defp emit_stop_event(id, start_time_mono, dataloader) do
+    :telemetry.execute(
+      [:absinthe, :middleware, :dataloader, :stop],
```

Comment:
Dataloader is technically orthogonal to  Absinthe. Let's use `:dataloader` as the top level key here not Absinthe. I think the overall key structure would be something like:
```
[:dataloader, :batches, :run, :start]
```

Reply:
It has been done :)